assets/templates/partials/filter.tmpl

```diff
@@ -137,6 +139,108 @@
     </noscript>
 </form>
 
+{{ if $population_types }}
+<form id="filterForm" method="get" class="js-auto-submit__form">
```

Comment:
You've got a few of the same form id here - is there a reason for this? Or should they be part of the same form?

Reply:
Same question to the hidden query input below.

Reply:
I assume the hidden inputs are left over from the days where JS might not have been required, but I checked whats on prod and it requires JS so removing.